examples/protocol_buffers/BUILD.bazel

```diff
@@ -5,6 +5,7 @@ load("@npm//@bazel/rollup:index.bzl", "rollup_bundle")
 load("@npm//@bazel/terser:index.bzl", "terser_minified")
 load("@npm//@bazel/typescript:index.bzl", "ts_config", "ts_devserver", "ts_library")
 load("@npm//http-server:index.bzl", "http_server")
+load("@rules_proto//proto:defs.bzl", "proto_library")
```

Comment:
was this just a warning you're fixing or is it related to the change?

Reply:
🤔  where did that symbol come from before I added that load?